package.json

```diff
@@ -14,37 +14,37 @@
         "lint": "npx eslint . --ext .js,.jsx,.ts,.tsx"
     },
     "devDependencies": {
-        "@aws-cdk/assert": "1.104.0",
+        "@aws-cdk/assert": "1.108.1",
```

Comment:
Let's not upgrade CDK in this PR. We have a separate process for upgrade and CDK is changing every week. We have documentation referencing this version so it should match. Generally a separate PR is used for that and we have a process.

Reply:
I'll removed version v1_20 of kubernetes and reset to 1.104.0